docs/src/4_dispatch/exercise.md

```diff
@@ -166,14 +166,11 @@ predict(x, 1000, 300)
 a prediction of income with the respective measurement error.
 
 
-Hopefully, this serves a a nice illustration of what multiple dispatch together with julias type system is able to achieve.
-When you wrote the `predict` function at the beginning of this workshop, you probably had no idea what multiple dispatch even is.
-But because we are able to define methods for important operations (like `*`, `+`), we can use any function or algorithmn that is composed of these operations.
+Hopefully, this serves as a useful illustration of what multiple dispatch with julias type system is able to achieve.
+Because we are able to define methods for important operations (like `*`, `+`), we can use any function or algorithmn that is composed of these operations.
 This is a very powerful idea that allows for great extensibility and interoperability of different packages.
-If you don't believe me, imagine somebody has written an `R` package for linear regression.
-Now you are in the situation that you have to deal with measurement error.
-I believe it would be impossible to get the published `R` package to work with measurement error without rewriting the whole package.
+For instance, imagine somebody has written an `R` package for linear regression that doesn't include measurement errors; it would likely be impossible to get it to work with measurement error without rewriting the whole package, while this could be relatively easily added to it in julia.
```

Comment:
```suggestion
For instance, imagine somebody has written an `R` package for linear regression that doesn't include measurement errors; it would likely be impossible to get it to work with measurement error without rewriting the whole package. 
However, as we demonstrated in this workshop, this is easily done in julia: When you wrote the `predict` function at the beginning of this workshop, you probably did not think about multiple dispatch or measurement error at all - but the code you wrote was still extensible, and we were later able to make it work with measurement error. This is the true power of julia - you don't have to think about extensibility too hard, just write normal julia code, and the rest is done by design of the language.
```